zokrates_cli/src/ops/mpc/beacon.rs

```diff
@@ -0,0 +1,152 @@
+use crate::constants::MPC_DEFAULT_PATH;
+use clap::{App, Arg, ArgMatches, SubCommand};
+use phase2::parameters::MPCParameters;
+use std::fs::File;
+use std::io::{BufReader, BufWriter};
+use std::path::Path;
+
+pub fn subcommand() -> App<'static, 'static> {
+    SubCommand::with_name("beacon")
+        .about("Applies a random beacon")
+        .display_order(3)
```

Comment:
What does this do? If it's necessary, we should have a comment explaining what it does

Reply:
It makes a strict order of commands in the help section when you run `zokrates mpc --help`. I made it so it is displayed in the order it should be run

Reply:
Right, see comment lower down about moving this to the mpc level